Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -543,7 +550,7 @@ extension SwiftLanguageService {
       let replacement: String
     }
 
-    guard let (preEditSnapshot, postEditSnapshot, edits) = self.documentManager.edit(notification) else {
+    guard let (preEditSnapshot, postEditSnapshot, edits) = orLog("Getting document manager", { try self.documentManager.edit(notification) }) else {
```

Comment:
CI says this line exceeds the maximum line lengths 😢 . Could you run swift-format? https://github.com/swiftlang/sourcekit-lsp/blob/main/CONTRIBUTING.md#formatting

Reply:
Just did